### ISSUE TITLE
Warum nicht über HA?

### ISSUE THREAD

**Issue by Roemer** [MrIceman11/e3dc-to-grafana#1]
Da du ja nun alle Daten im Home Assistant hast, warum dann nicht auch gleich von  dort ein Prometheus Export (nach VictoriaMetrics) und dann das direkt im Grafana? Dann hast du grad auch die daily/weekly/monthly Energy usage und alles. Dürfe ein viel einfacheres Setup sein.

**Comment by MrIceman11** [comment]
Jo, hab ich mir auch schon gedacht. Werde ich auch mal in Angriff nehmen.

**Comment by Roemer** [comment]
Kann dir gerne was Konfigs geben wenn du was brauchst. Aber ist relativ einfach. Prometheus-Export aus HA aktivieren, in VictoriaMetrics als Target (Scraper) hinuzufügen und das wars eigentlich schon.

**Comment by MrIceman11** [comment]
Ja hab ich grade auch schon gesehen. Wenn ich Zeit dafür habe, werde ich das definitiv umsetzen.
